mq/main/pom.xml

```diff
@@ -165,7 +165,7 @@
             <dependency>
                 <groupId>org.glassfish</groupId>
                 <artifactId>jakarta.json</artifactId>
-                <version>2.0.0-RC2</version>
+                <version>2.0.0</version>
```

Comment:
In GF [appserver/pom.xml](https://github.com/eclipse-ee4j/glassfish/blob/d6f0dfa7adf823908d95bfc3e9490ec244d509ee/appserver/pom.xml#L142)